leshan-core/src/main/java/org/eclipse/leshan/util/Key.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.leshan.util;
```

Comment:
`util` package? why not moving it to`org.eclipse.leshan.server.californium.impl` if it used only there.